index.html

```diff
@@ -78,5 +80,221 @@ <h2 class="secondary-header">LET’S CONNECT</h2>
+    <section class="works">
+      <!-- first project  -->
+      <div class="projects container">
+        <img class="project-img" src="images/works/twitter-shot.jpg" alt="" />
+        <h3 class="tertiary-heading">Tonic</h3>
+        <ul class="project-details">
+          <li>CONOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+        <p class="project-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="project-lang">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <button class="project-btn">See project</button>
+      </div>
+      <!-- second project  -->
+      <div class="projects container">
+        <img class="project-img" src="images/works/github-shot.jpg" alt="" />
+        <h3 class="tertiary-heading">Multi-Post Stories</h3>
+        <ul class="project-details">
+          <li>CONOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+        <p class="project-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="project-lang">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <button class="project-btn">See project</button>
+      </div>
+      <!-- thirs project  -->
+      <div class="projects container">
+        <img class="project-img" src="images/works/shot3.png" alt="" />
+        <h3 class="tertiary-heading">Tonic</h3>
+        <ul class="project-details">
+          <li>CONOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+        <p class="project-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="project-lang">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <button class="project-btn">See project</button>
+      </div>
+      <!-- fourth project  -->
+      <div class="projects container">
+        <img class="project-img" src="images/works/twitter-shot.jpg" alt="" />
+        <h3 class="tertiary-heading">Multi-Post Stories</h3>
+        <ul class="project-details">
+          <li>CONOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+        <p class="project-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="project-lang">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <button class="project-btn">See project</button>
+      </div>
+    </section>
+    <!-- ABOUT ME SECTION  -->
+    <section class="about container">
+      <h2 class="secondary-header">About Myself</h2>
```

Comment:
- [ ] Please add a flexbox to this `about section` because this is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md). The content of the `About myself` section is aligned with Flexbox. But you are not using any flexbox in this about section :slightly_smiling_face: